public/app/features/live/live.ts

```diff
@@ -133,15 +144,17 @@ class CentrifugeSrv implements GrafanaLiveSrv {
       },
     };
     c.subscription = this.centrifuge.subscribe(path, callbacks);
+    return connected;
   }
 
   getChannelStream<T>(path: string): Observable<T> {
```

Comment:
The original idea is that a few things may connect to the same stream -- in an alternate PR, I tried:

https://github.com/grafana/grafana/pull/27583/files#diff-234f06c46c08d978307cf6c087712a55R134

Reply:
that should still be possible I think